infraestructure/services/mapper.go

```diff
@@ -0,0 +1,40 @@
+package services
+
+import (
+	"github.com/alexis-aguirre/golang-bootcamp-2020/domain/model"
+)
+
+func lyricResponseToSong(lyricResponse *HappiLyricResponse) *model.Song {
+	song := &model.Song{}
+	song.ID = lyricResponse.Result.IDTrack
+	song.Name = lyricResponse.Result.Track
+	song.AlbumID = lyricResponse.Result.IDAlbum
+	song.Album = lyricResponse.Result.Album
+	song.Interpreter = lyricResponse.Result.Artist
+	song.InterpreterID = lyricResponse.Result.IDArtist
+	song.Lyric = lyricResponse.Result.Lyrics
+	return song
+}
+
+func seachResponseToSongSlice(searchResponse *HappiSearchResponse) []*model.Song {
+	var songs []*model.Song
+	for i := 0; i < searchResponse.Length; i++ {
+		currentResult := searchResponse.Result[i]
```

Comment:
```suggestion
	for _, currentResult := range searchResponse {
```

What do you think about this?